pkg/services/govmomi/vcenter/clone.go

```diff
@@ -167,6 +167,12 @@ func Clone(ctx context.Context, vmCtx *capvcontext.VMContext, bootstrapData []by
 		memMiB = 2048
 	}
 
+	var memLockedToMax *bool
+	if vmCtx.VSphereVM.Spec.MemoryReservationLockedToMax != nil {
+		memLockedToMax = new(bool)
```

Comment:
```suggestion
```

That should not be required right?

Reply:
Yep. This is only required while using pass through devices and thats already in the code.

Reply:
So let's remove it?

Reply:
obsolete with the below.